compiler/record-minmax-conversion-test/gen_h5_random_inputs.py

```diff
@@ -39,8 +39,8 @@
 
     for j in range(len(input_details)):
         input_detail = input_details[j]
-        input_data = np.array(
-            np.random.random_sample(input_detail["shape"]), input_detail["dtype"])
+        input_data = np.array(10 * np.random.random_sample(input_detail["shape"]) - 5,
```

Comment:
It would be better to add comment that shows the range of its random data. Maybe [-5, 5]?

Reply:
Added comment. Thanks.